scss/application.scss

```diff
@@ -0,0 +1,4 @@
+.title {
+    color: red;
+  }
+  
```

Comment:
I think you have an unintended space, check that and remember to leave an EOF new line :smile: